### PR TITLE
Add localStorage unique value for feedback form

### PR DESCRIPTION
**Pull request by tom-fleming**
**Description:**
This PR adds a local storage value when users submit the feedback form. This value is intended to be unique, and remain set and consistent across new browser tabs and windows, and is passed to the Formstack submission.

**Jira:** (Skip unless you are MA staff)
DP-27259

**To Test:**
If you have access to Formstack:
- [ ] Go to a page, such as /governor, select yes or no, enter "test feedback" in the body, and submit
- [ ] In Formstack, go to your test submission, and confirm that mg_unique_id has a value. Make note of this value.
- [ ] Go to another page, such as /rmv, select yes or no, enter "test feedback" in the body, and submit
- [ ] In Formstack, go to your test submission, and confirm that mg_unique_id has a value and that it is the same value as your first test submission
- [ ] Steps can be repeated for new windows and tabs (optional)

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: docroot/themes/custom/mass_theme/overrides/js/device.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  Drupal.behaviors.massgovDevice = {
+    attach: function (context) {
+      once('massgovDeviceId', 'html', context).forEach(function() {
+        if (window.crypto && localStorage.getItem('massgovDeviceId') === null) {
+          localStorage.setItem('massgovDeviceId', window.crypto.randomUUID());
+        }
+      });
+    }
+  };
+})(jQuery);
```
Comment (justafish) [inline docroot/themes/custom/mass_theme/overrides/js/device.js]:
```suggestion
(function (Drupal, once) {
  Drupal.behaviors.massgovDevice = {
    attach: function (context) {
      once('massgovDeviceId', 'html', context).forEach(function() {
        if (window.crypto && localStorage.getItem('massgovDeviceId') === null) {
          localStorage.setItem('massgovDeviceId', window.crypto.randomUUID());
        }
      });
    }
  };
})(Drupal, once);
```

jQuery not needed here :smile_cat:

**Review by justafish — CHANGES_REQUESTED** [review]
I don't have access to formstack, but I do get the value in local storage as expected :+1:

**Comment by justafish** [comment]
validate test is failing ^

File: docroot/themes/custom/mass_theme/overrides/js/device.js (hunk 1 above)
Comment (tom-fleming) [reply docroot/themes/custom/mass_theme/overrides/js/device.js]: @justafish Ah, thanks! Old habits die hard

**Comment by tom-fleming** [comment]
> validate test is failing ^

@justafish I fixed the errors in this commit: https://github.com/massgov/openmass/pull/1913/commits/1c1ac9264cab4279efc2b17a6e66acb0f094bbd7. Some of the changes I had to make here are the result of us using .eslintrc.legacy.json from Drupal core, which meant that `once` was not included as a global and nullish coalescing appears to not be supported. I wrote a note in the frontend D10 upgrade ticket about maybe updating this: https://massgov.atlassian.net/browse/DP-27296?focusedCommentId=308712

**Review by justafish — APPROVED** [review]
lgtm! :+1:

**Pull request merged**